portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,61 @@
-#content {
+@keyframes changebg {
+    from {background-color: #6A17E8}
+    to {background-color: white}
```

Comment:
Missing semi-colons. Example :
from {background-color: #eee;}
to {background-color: #fff;}